src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureCount = 5;
+        int rnd = random.nextInt(figureCount);
+        int valueCount = 100;
```

Comment:
5 and 100 should be [constants](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)